lib/clients/ai/openai/gpt3_5.ts

```diff
@@ -42,7 +42,17 @@ export class GPT3_5 extends GPTAIModel {
         tokens: this.calculateTokenMultiplier(tokens),
       };
     } catch (error) {
-      log.warn(`GPT3.5 completion ${log.vars({ error: error?.response ?? error, messages, params })})}`);
+      const truncatedMessages = messages.slice(0, 10).map(({ content, ...rest }) => ({
+        ...rest,
+        content: content.substring(0, 200),
+      }));
+      log.warn(
+        `GPT3.5 completion ${log.vars({
+          error: error?.response ?? error,
```

Comment:
Can `error` just be `error.message`? And maybe add the code too?

```suggestion
          error: error?.message,
          code: error?.code,
```

Reply:
Sure